### PR TITLE
Cleaned up use of IMAGE_PULL_POLICY

### PR DESCRIPTION
**Pull request by vbusch**
Signed-off-by: Vanessa <vbusch@redhat.com>

### CODE REVIEW

File: address-space-controller/src/main/resources/templates/standard-space-infra.yaml [hunk 1]
```diff
@@ -462,8 +462,6 @@ objects:
             value: ${BROKER_PLUGIN_IMAGE}
           - name: TOPIC_FORWARDER_IMAGE
             value: ${TOPIC_FORWARDER_IMAGE}
-          - name: ${IMAGE_PULL_POLICY}
```
Comment (k-wall) [inline address-space-controller/src/main/resources/templates/standard-space-infra.yaml]: I think the defect is that "- name: ${IMAGE_PULL_POLICY}" should read "- name: IMAGE_PULL_POLICY".  The purpose is to propagate the IMAGE_PULL_POLICY down to the resources created from the templates.

File: address-space-controller/src/main/resources/templates/standard-space-infra.yaml [hunk 2]
```diff
@@ -545,7 +543,7 @@ objects:
                 key: cookie-secret
                 optional: true
           image: ${AGENT_IMAGE}
-          imagePullPolicy: ${env.IMAGE_PULL_POLICY}
```
Comment (k-wall) [inline address-space-controller/src/main/resources/templates/standard-space-infra.yaml]: I think your change here is incorrect.

**Comment by k-wall** [comment]
@vbusch can you review please?

**Comment by vbusch** [comment]
LGTM

**Pull request merged**